opensrp-app/src/main/java/org/smartregister/SyncConfiguration.java

```diff
@@ -236,4 +236,13 @@ public boolean validateUserAssignments() {
     public boolean skipUnsyncedTasksOnFetchFromServer(){
         return false;
     }
+
+    /**
+     * Specifies whether to trigger search after each character is typed (return true). Alternatively return false after clicking
+     * a button or pressing the Return key.
+     * @return {@link Boolean}
+     */
+    public boolean fastPatientRegisterSearch(){
```

Comment:
I think we should move this configuration to use the `app.properties` implementation instead since this class is mostly for configurations of the Sync functionality . See [sample app.properties file](https://github.com/opensrp/opensrp-client-core/blob/master/sample/src/main/assets/app.properties)

Also check out the README section on [Configurability](https://github.com/opensrp/opensrp-client-core#configurability)